cabal.project

```diff
@@ -1,4 +1,6 @@
 packages: .
+          https://hackage.haskell.org/package/filepath-2.0.0.0/candidate/filepath-2.0.0.0.tar.gz
+
```

Comment:
I am not entirely fond of manifest remote dependencies.  Ideally a build should be able to complete entire with locally staged inputs.  Is there some other way to signal the `file path` dependency?  Is this just a temporary measure?  When will this be resolved?

Reply:
This will be resolved once I actually publish filepath-2.x... that requires a few minor issues to be resolved: https://gitlab.haskell.org/haskell/filepath/-/issues?milestone_title=2.0.1

Reply:
Understood. I wouldn't want  to make a "release" with this unaddressed (unless unavoidable), but merging this as an interim commit, that could be used in a submodule, ... is fine if the consuming package deems it acceptable.  Does this work for you?

Any thoughts on the other comments?

Reply:
My plan is to make this PR mergable and then have it be merged after the filepath release, following a unix and Win32 release. The new filepath release cannot really be used without updated unix/Win32 releases.

Reply:
So, I take it the plan is to resolve all the issues, and then wait for the file path release?  You'll then ping the "unix" maintainers to push the button?

Reply:
Yes, precisely.